dev_capabilities.json

```diff
@@ -37,6 +37,10 @@
             "pattern" : "^/halp",
             "url" : "http://imgur-hipchat-dev.herokuapp.com",
             "event" : "room_message" } ],
+            {
+            "pattern" : "^/roll .+",
+            "url" : "http://imgur-hipchat.herokuapp.com",
```

Comment:
the `dev_capabilities.json` file requires that all hostnames point to the staging env: `imgur-hipchat-dev`, otherwise the web hook will be sent to the wrong place(!)